app/src/main/res/layout/content_main.xml

```diff
@@ -54,6 +52,18 @@
             app:circle_start_color="@android:color/holo_purple"
             app:dots_primary_color="@android:color/holo_blue_dark"
             app:dots_secondary_color="@android:color/holo_blue_light"
-            app:icon_size="25dp" />
+            app:icon_size="22dp" />
+
+        <com.like.LikeButton
+            android:id="@+id/vector_android_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:circle_end_color="@android:color/holo_green_dark"
+            app:circle_start_color="@android:color/holo_green_dark"
+            app:dots_primary_color="@android:color/holo_green_dark"
+            app:dots_secondary_color="@android:color/holo_green_light"
+            app:icon_size="22dp"
+            app:like_drawable="@drawable/ic_android_green_500_24dp"
+            app:unlike_drawable="@drawable/ic_android_grey_500_24dp" />
```

Comment:
This is vector drawable sample.

<img width="367" alt="slack" src="https://cloud.githubusercontent.com/assets/1269214/13723921/1c058194-e8b7-11e5-9416-565232e23ad6.png">